backend/app/migrations/0001_initial.py

```diff
@@ -17,13 +17,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('identifier', app.fields.LowercaseCharField(max_length=40)),
-                ('type', models.CharField(choices=[
-                        ('subj', 'Subject'),
-                        ('obj', 'Object'),
-                        ('pos_det', 'Possessive determiner'),
-                        ('pos_pro', 'Possessive pronoun'),
-                        ('reflex', 'Reflexive')
-                    ], max_length=7)),
+                ('type', models.CharField(choices=[('subj', 'Subject'), ('obj', 'Object'), ('pos_det', 'Possessive determiner'), ('pos_pro', 'Possessive pronoun'), ('reflex', 'Reflexive')], max_length=7)),
             ],
         ),
-    ]
+    ]
```

Comment:
This looks like a simple reformatting caused by some copying and pasting or something, but for the sake of setting the pattern, it's best if we don't modify old migrations. Would it be possible to back out this change? Or did it occur automatically for some other reason?

Reply:
I don't remember intentionally modifying that migration file--maybe it occurred automatically or when I was reconstructing the order of migrations. I can go back and revert this change.